STARTERKIT/gulpfile.js

```diff
@@ -118,31 +119,23 @@ options.eslint = {
 // Gulp watch. Since polling is less efficient, we disable polling by default.
 options.gulpWatchOptions = {};
 // options.gulpWatchOptions = {interval: 1000, mode: 'poll'};
-
-
-// ################################
 // Load Gulp and tools we will use.
-// ################################
 var gulp      = require('gulp'),
   $           = require('gulp-load-plugins')(),
   browserSync = require('browser-sync').create(),
   del         = require('del'),
   // gulp-load-plugins will report "undefined" error unless you load gulp-sass manually.
-  sass        = require('gulp-sass'),
-  kss         = require('kss'),
-  cache       = require('gulp-cached');
```

Comment:
Better to move inline comment above other definitions to keep all indented similar